### DISCUSSION TITLE
Tab Width Closing Behavior

### DISCUSSION THREAD

**Discussion by Christian-Kouts** [sumatrapdfreader/sumatrapdf#2406] [Ideas]
Every time I'm mass closing PDF tabs I wish the behaviour was similar to the likes of my web browser Chrome. In Chrome, when closing tabs the tab size remains the same such that you do not have to move the mouse. Briefly, after closing your last tab, the tab width is then finally adjusted to accommodate the now lesser amount of tabs.

**Comment by GitHubRulesOK** [comment]
my browser does same as sumatraPDF more tabs get smaller, there is tab width setting so if small that can help keep more constant short tabs and with newer tooltip names less need for wide tabs,

**Reply by Christian-Kouts** [reply]
When the width of the tabs are < default width (maybe your Sumatra window width is small or maybe you have many tabs open), as you begin mass closing the tabs, the tabs dynamically change width after every closed tab. When in reality (how the tab width works in Chromium and many other softwares) the dynamic tab-width should only change roughly 1 second after your last closed tab to allow you to mass close tabs without having to re-position the mouse over the tab close "X" button. I can post a gif of the ideal behaviour if you still are unclear.

**Comment by kjk** [comment]
The problem is that it's a "nice to have" feature that would require a lot of effort to implement given technology we have at our disposal (win32 tab control). Therefore it's low on my priority list.

**Reply by Christian-Kouts** [reply]
Totally understand. Provided millions of people use Sumatra, maybe it could be higher on the priority list.

**Reply by GitHubRulesOK** [reply]
several millions of those users paid for clone copies but their retailers don't contribute to the needed resources !!
